### PR TITLE
feat: add the ability to set seeds for ducks

### PR DESCRIPTION
**Pull request by vcokltfre**
/duck and /manduck both now support a seed query param to use a seed to generate a duck. The seed is an integer, for example `GET /duck?seed=1234`

This also fixes the fact that quackstack wont start because the dockerfile was using an outdated location for the main file.

### CODE REVIEW

**Review by Kronifer — APPROVED** [review]
Looks good!

File: quackstack/ducky.py [hunk 1]
```diff
@@ -31,7 +31,8 @@ class DuckBuilder:
         filename.stem: filename for filename in (ASSETS_PATH / "accessories/outfits").iterdir()
     }
 
-    def __init__(self):
+    def __init__(self, seed: int = None):
+        self.random = Random(seed) if seed else Random()
```
Comment (doublevcodes) [inline quackstack/ducky.py:35]: Can't you just leave this as `Random(seed)`? The docs say the seed can be `NoneType`.

File: quackstack/manducky.py [hunk 2]
```diff
@@ -41,7 +41,8 @@ class ManDuckBuilder:
         }
     }
 
-    def __init__(self):
+    def __init__(self, seed: int = None):
+        self.random = Random(seed) if seed else Random()
```
Comment (doublevcodes) [inline quackstack/manducky.py:45]: Same here

File: quackstack/ducky.py [hunk 3]
```diff
@@ -31,7 +31,8 @@ class DuckBuilder:
         filename.stem: filename for filename in (ASSETS_PATH / "accessories/outfits").iterdir()
     }
 
-    def __init__(self):
+    def __init__(self, seed: int = None):
```
Comment (doublevcodes) [inline quackstack/ducky.py:34]: This should probably be typehinted with `typing.Optional`

File: quackstack/manducky.py [hunk 4]
```diff
@@ -41,7 +41,8 @@ class ManDuckBuilder:
         }
     }
 
-    def __init__(self):
+    def __init__(self, seed: int = None):
```
Comment (doublevcodes) [inline quackstack/manducky.py:44]: `typing.Optional` also applied here

File: api/main.py [hunk 5]
```diff
@@ -58,15 +58,15 @@ async def get_duck(duck: Optional[DuckRequest] = None) -> DuckResponse:
 
 
 @app.get("/manduck", response_model=DuckResponse)
-async def get_man_duck(manduck: Optional[ManDuckRequest] = None) -> DuckResponse:
+async def get_man_duck(manduck: Optional[ManDuckRequest] = None, seed: int = None) -> DuckResponse:
```
Comment (doublevcodes) [inline api/main.py:61]: Is this not optional? `typing.Optional`

**Review by doublevcodes — COMMENTED** [review]
Let me know if I've totally misunderstood what's happening here.

**Review by doublevcodes — APPROVED** [review]
:shipit:

**Review by jb3 — APPROVED** [review]

**Pull request merged**